internal/cmd/config/helptext/preferences.txt

```diff
@@ -12,6 +12,10 @@
 │ endpoint         │ Hetzner Cloud API    │ string      │ endpoint         │ HCLOUD_ENDPOINT         │ --endpoint      │
 │                  │ endpoint             │             │                  │                         │                 │
 ├──────────────────┼──────────────────────┼─────────────┼──────────────────┼─────────────────────────┼─────────────────┤
+│ no-wait          │ If true, do not wait │ boolean     │ no_wait          │ HCLOUD_NO_WAIT          │ --no-wait       │
```

Comment:
I prefer `wait`.

For example, if the user set this to `true` in their config file, but want to wait for one specific call, they would need to set `--no-wait=false` which is a double negative and confusing.

Reply:
Should we add general support for `--no-[bool-option]` flags?

Reply:
Since we have multiple source of configuration, I think having a negative flag for our bool configuration is indeed handy.

For example if we have --wait, we should have --no-wait, or --include-deprecated-architecture and --no-include-deprecated-architecture

Reply:
> This is something for a different PR then. So we just go for `--wait=false` now?

I would prefer that, yes.